internal/metamorph/zmq.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 	"time"
 
+	metamorph_p2p "github.com/bitcoin-sv/arc/internal/metamorph/bcnet/p2p"
```

Comment:
Why not just use `p2p` in places where there's no other p2p other than metamorph_p2p?

Reply:
After discussion, let's go back to the previous names `metamorph_p2p` and `blocktx_p2p`